CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-[...]
+- [BREAKING CHANGE] Rename Notification to Snackbar
```

Comment:
`[BREAKING CHANGE] Rename Notification to Snackbar` > `` **[BREAKING CHANGE]** Rename `Notification` to `Snackbar` ``